android/src/main/java/com/pdftron/reactnative/viewmanagers/DocumentViewViewManager.java

```diff
@@ -516,6 +516,11 @@ public void setHighlighterSmoothingEnabled(DocumentView documentView, boolean hi
         documentView.setHighlighterSmoothingEnabled(highlighterSmoothingEnabled);
     }
 
+    @ReactProp(name="maxSignatureCount")
+    public void setMaxSignatureCount(DocumentView documentView, int maxSignatureCount){
+        documentView.setMaxSignatureCount(maxSignatureCount);
```

Comment:
can avoid this call and call `SignatureDialogFragment.MAX_SIGNATURES = maxSignatureCount;` directly here because it's static